nix/rust-package.nix

```diff
@@ -91,7 +95,16 @@ let
   };
 
   sharedArgs =
-    if runTests then sharedArgsBase // {
+    if runCoverage then sharedArgsBase // {
+      cargoTestExtraArgs = "--workspace -F runtime-async-std -F runtime-tokio";
```

Comment:
Only `runtime-async-std` should be used

Reply:
No, we have tests which depend on the tokio flag too.

Reply:
Then we should rewrite the tests iun terms of tokio only and obsolete async-std for now.